### PR TITLE
Exception handling - Part1

### PR DESCRIPTION
**Pull request by Ritabrata1080**
- [x] File Header updated with proper description
- [x] Code includes proper formatting and necessary logs added
- [x] No new warning generated by PR
- [x] CodeQL scan passed

### CODE REVIEW

File: src/ExceptionHandling.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        // Since in this case we know it will throw an Arithmetic exception in try block, so
+        // we are catching the exception through ArithmeticException
+        catch (ArithmeticException ae) {
+            System.out.println(ae.getMessage());
+        }
+        // This catch block will catch generic exception.
+        // It is used when we don't know the possible cause for exception
+        catch (Exception e) {
+            // getMessage() will only give the reason of exception without any class details or stackTrace
+            //System.out.println("Cannot perform division : " + e.getMessage());
```
Comment (avanura23) [inline src/ExceptionHandling.java:20]: Is this supposed to be in the code or commented?

File: src/ExceptionHandling.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/*
+ Developed by Ritabrata, 2024
+ */
+
+
+class performOperations {
```
Comment (avanura23) [inline src/ExceptionHandling.java:6]: Class name should with a capital letter

Comment (Ritabrata1080) [reply src/ExceptionHandling.java]: Addressed

File: src/ExceptionHandling.java (hunk 1 above)
Comment (Ritabrata1080) [reply src/ExceptionHandling.java:20]: It's a good practice not to keep commented code, but this is for reference purpose. We can keep as of now. In future, will create a PR template where we need to remove all commented code if it's not required.

Comment (avanura23) [reply src/ExceptionHandling.java:20]: Okay

**Review by avanura23 — APPROVED** [review]

**Pull request merged**